tools/circle_plus_gen/how-to-write-tparam-json.md

```diff
@@ -0,0 +1,77 @@
+# How to write Hyperparameters JSON file
+
```

Comment:
I'm not sure this kind of document is understandable and useful. 
So, I'm open to any opinion. Please don't hesitate to share your idea or reviews.

Reply:
(background) 
I tried to write things - by agreeing this comment (https://github.com/Samsung/ONE/pull/13012#issuecomment-2119522242)